Tiltfile

```diff
@@ -102,6 +102,12 @@ if os.getenv('INITIUM_DEPLOY_MINIMAL', 'false') == 'false':
 
     # ===== Tilt local resources =====
 
+    local_resource(
+        'grafana-password',
+        cmd='kubectl get secret -n grafana grafana-initial-admin-secret -o jsonpath="{.data.admin-password}" | base64 -d; echo',
```

Comment:
Could we give the name `grafana-initial-admin-secret` in a single location? We currently have this in 3 places. Maybe from Makefile?

Reply:
Although it would be a nice improvement, I see two problems with that approach:
- One of the occurrences is the name of an attribute in a Helm chart's values file, which has to be a valid YAML file without any kind of templating. This refactor would require extra effort to implement some preprocessing.
- Another occurrence is a value in a Helm chart's template. Although we can use Helm templating to solve this, it would also require extra effort to create the data source for this value.

Therefore, I believe that the investment in effort far outweighs the potential return.